opendbc/car/toyota/carcontroller.py

```diff
@@ -70,6 +71,7 @@ def update(self, CC, CS, now_nanos):
       if CS.secoc_synchronization['RESET_CNT'] != self.secoc_prev_reset_counter:
         self.secoc_lka_message_counter = 0
         self.secoc_lta_message_counter = 0
+        self.secoc_acc_message_counter = 0
```

Comment:
Can these all be using the same counter?

Reply:
Good question. My initial hunch is to say no as I would assume the car is keeping track of message counts per each secoc signed message. But this is only a hunch so I can validate by making them all use the same counter and see if it breaks. Will report back to you after I test this out.

Reply:
Ah, I see it's because LKA and LTA are 100 Hz and 50 Hz respectively. You could technically use one counter and then just int divide it by 2 for LTA:

```
frame = 0
for i in range(100):
  frame += 1
  if i % 2 == 0:
    print(frame // 2)
```

Reply:
Good point on the frequency. If it wasn’t for the 33 Hz ACC_CONTROL your suggestion would have been a nice simplification. For my own curiosity could you point me to where the message frequency would be documented/defined?

Reply:
cabana shows the frequency of each message when you open the route